### ISSUE TITLE
version of python and numpy

### ISSUE THREAD

**Issue by paskino** [DiamondLightSource/Savu#868]
What are the versions of numpy and python that are required by Savu?

I seem to see Python 3.7 and NumPy 1.15, which seem pretty old.

**Comment by dkazanc** [comment]
that's correct Python 3.7 and NumPy 1.15. There is a plan to move to Py 3.9 + numpy 1.20, however it results in hdf5 problems due to upgraded h5py library.

**Comment by DimitriosBellos** [comment]
Hi Daniil, 

Since you will move to Python 3.9 will savu be compatible with Python 3.8 too? Also is there an update for the version transition?

Thanks
Dimitrios
